src/OSPSuite.UI/Extensions/ChartControlExtensions.cs

```diff
@@ -99,6 +110,8 @@ public static void CopyToClipboard(this UxChartControl chartControl, string wate
          {
             copyFontAndSizeSettings(chartControl, cloneOfChartControl);
             AddWatermark(cloneOfChartControl, watermark);
+            prepareChartForCopying(cloneOfChartControl);
```

Comment:
I am wondering why you are not doing this line on line 104
In fact, I think this should be changed a bit. Why do we have something like this?
`chartControl.CopyChartToClipboard(chartControl);`

It should be `chartControl.CopyChartToClipboard();`

and why do we have
`chartControl.CopyChartToClipboard(cloneOfChartControl);`

it should be
`cloneOfChartControl.CopyChartToClipboard();`

So somehow now that we are modifying the chart, we always need to clone it, update font and settings etc from the clone, and call this method (prepareChartForCopying)

And we need to verify for barcharts to make sure this works as expected. (as there was some issue it seems with devbepxress at some point, maybe the bug is corrected)

Reply:
I have changed 
`chartControl.CopyChartToClipboard(cloneOfChartControl);`

to 
`cloneOfChartControl.CopyChartToClipboard();`

this was actually written a bit weird till now, I agree. 

I have also changed the code for this to also be done if we do not set the watermark - this second function here is only being used for PI Time Analysis and Sensitivity analysis, this is why I did not find out the change was not working without watermark for these cases.

Reply:
@msevestre so now we need to create the clone anyways, since we need to change the border of the image. 

tbh the version of the function for the ChartView (line 37), does not check if the watermark is null or empty, and still works fine without a watermark. If you agree I can remove the check, since it seems redundant.

Regarding the barcharts that are also mentioned in the ToDo, I am not sure how to reproduce this or what exactly the bug is...let's talk about this (also maybe solve in a separate issue )

Reply:
Bar chart are in PKSIm
![image](https://user-images.githubusercontent.com/1041237/164724901-b3b48cce-2630-4d48-aa6e-600c7616cb22.png)

For this, there is no watermark.

Reply:
Line 37 has a Chart as parameter. and there, there is always a Watermark. The other plot is independent from the Chart (IChart instance) not the control

If you can try with PKSim and we see that it works again, then we can remove OTherwise we need to keep

Reply:
@msevestre so the watermark part ( when no watermark is specified) is being handled in AddWatermark function, in lines124 and then 130-131. So I will remove this check from both functions, it is not necessary.